### PR TITLE
test(modeling): validate poly3 are coplanar

### LINKED ISSUE #1059: Possible flaw in poly3.validate
## Expected Behavior
Perhaps, poly3.validate should check to see that the vertices are coplanar?

## Steps to Reproduce the Problem

  1.
  2.
  3.

## Specifications

  - Version:
  - Platform:
  - Environment: (browser, local server, node.js etc)

### PR DESCRIPTION
**Pull request by platypii**
Resolves #1059. This adds additional validation to poly3 to check that vertices are coplanar. Thanks @briansturgill!

The minimum error NEPS = 1e-13 was chosen because the minimum value needed for tests to pass was 4e-15. NEPS of 1e-13 keeps a tight bound, but still allowing reasonable floating point error.

Edit: Also moved NEPS into maths constants.

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Does your submission pass tests?

### CODE REVIEW

File: packages/modeling/src/geometries/poly3/validate.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
+const signedDistanceToPoint = require('../../maths/plane/signedDistanceToPoint')
 const vec3 = require('../../maths/vec3')
 const isA = require('./isA')
 const isConvex = require('./isConvex')
 const measureArea = require('./measureArea')
+const plane = require('./plane')
+
+const NEPS = 1e-13
```
Comment (z3dev) [inline packages/modeling/src/geometries/poly3/validate.js:8]: Maybe it's time to move this to maths/constants.js... we have multiple copies everywhere. :(

```
primitives/triangle.js:const NEPS = 1e-13
utils/trigonometry.js:const NEPS = 1e-13
maths/utils/aboutEqualNormals.js:const NEPS = 1e-13
operations/modifiers/mergePolygons.js:const NEPS = 1e-13
operations/modifiers/retessellate.js:const NEPS = 1e-13
```

**Review by z3dev — COMMENTED** [review]
@platypii what do you think about defining NEPS in maths/constants?

we can renovate the other code later in another PR.

File: packages/modeling/src/geometries/poly3/validate.js (hunk 1 above)
Comment (platypii) [reply packages/modeling/src/geometries/poly3/validate.js]: Moved NEPS into constants, and updated references to it :+1:

**Review by z3dev — DISMISSED** [review]
this will be a nice addition to the validate function. thanks.

**Comment by platypii** [comment]
@z3dev just realized I missed NEPS in the typescript file, pushed a commit with a fix

**Review by z3dev — APPROVED** [review]
Oh… just in time fix!

**Pull request merged**